docs/VSCODE_CONFIGURATION.md

```diff
@@ -0,0 +1,37 @@
+CONFIGURING VSCODE 
+==================
+
+In order to debug our flask app using VSCODE python debugger, follow the following steps:
+
+1. Install [VSCODE](https://code.visualstudio.com)
+2. In VSCODE Install the following extensions: Microsoft Python Extension](https://github.com/Microsoft/vscode-python), [Docker Extension](https://github.com/microsoft/vscode-docker), [Microsoft Remote Development](https://github.com/Microsoft/vscode-remote-release)
```

Comment:
There is a missing `[` before `Microsoft Python Extension]`
![image](https://user-images.githubusercontent.com/58947331/112679534-53a04280-8e7d-11eb-9b21-81476eff861e.png)